jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -45,3 +45,23 @@
         branch: master
         job-name: ci-kubernetes-e2e-gce-flaky
         repo-name: k8s.io/kubernetes
+    - kubernetes-e2e-gke-autoscaling:
+        branch: master
+        job-name: ci-kubernetes-e2e-gke-autoscaling
+        repo-name: k8s.io/kubernetes
```

Comment:
I explicitly do not want this sort of magic. It is more important to have an easy to understand config than a reduction in lines.